app/Http/Controllers/Chat/ChatController.php

```diff
@@ -123,65 +124,88 @@ public function __construct()
      */
     public function updates()
     {
-        $params = request()->all();
+        $request = request()->all();
+        $params = get_params($request, null, [
+            'channel_id:int',
+            'history_since:int',
+            'limit:int',
+            'since:int',
+        ], ['null_missing' => true]);
+
+        $includes = get_params($request, 'includes', [
```

Comment:
should this be this or `includes[]=messages&includes[]=presence` parsed above with `string[]` and use `Set` (with default everything when null/not specified/empty(?)) :thinking: